### PR TITLE
Chapter 31: Veteran info page

### PR DESCRIPTION
**Pull request by jbalboni**
I created a shared page for the vet info fields, since most of them are the same between the two forms.

screen shot 2018-02-20 at 4 53 20 pm

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]
good work

File: src/js/vre/pages/veteranInfo.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export default function createVeteranInfoPage(schema, extra) {
+  const {
+    veteranFullName,
+    veteranDateOfBirth,
+    veteranSocialSecurityNumber,
+    veteranVaFileNumber
+  } = schema.properties;
+
+  return {
+    title: 'Veteran Inforation',
```
Comment (cehsu) [inline src/js/vre/pages/veteranInfo.js]: Looks like we've got a typo here "Information"

**Review by cehsu — APPROVED** [review]
LGTM. Had a minor comment.

**Review by stacyhlinn — APPROVED** [review]
looks good! let's figure out the VA office later.

**Pull request merged**